operator/src/main/resources/scripts/utils.py

```diff
@@ -2,10 +2,15 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
 
 # Usage: trace('string')
+#        trace(logLevel,'string')
 #
-# This matches format of bash utils.sh trace, and the operator's log format.
+# Valid values for logLevel are SEVERE|WARNING|ERROR|INFO|CONFIG|FINE|FINER|FINEST
+#    'ERROR' is converted to 'SEVERE' 
```

Comment:
Do we need to keep the ERROR log level?

Reply:
I lean towards keeping it.  I kept it around as an extra precaution in case someone used it by accident.  It's just too easy to assume 'error' is a valid, and since bash isn't compiled such a mistake wouldn't be caught until runtime - which is too late for a rare error path...

Reply:
ok